CODE_OF_CONDUCT.md

```diff
@@ -1 +1,89 @@
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Being friendly and patient
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Trying to understand why we disagree
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+FaithTech is responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+We have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any individual for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Our Faith
+
+We will always be grounded in our faith and beliefs. Because of our beliefs we will always be open and loving:
+
+> Do everything in love
+> [1 Corithians 16:14 NIV](https://www.bible.com/bible/111/1co.16.14)
+
+> This is my command: Love each other.
+> [John 15:17 NIV](https://www.bible.com/bible/111/jhn.15.17)
+
+> We love because he first loved us.
+> [1 John 4:19 NIV](https://www.bible.com/bible/111/1jn.4.19)
+
+Faith is a journey and we want to support you wherever you are on that path. We understand if you do not share our beliefs and still welcome you to join us. We will not harrass you, but some of our activities may make you feel uncomfortable.
+
+Participation is voluntary?
+
+Unless otherwise stated, we are not here to debate or defend our beliefs. If you would like to do so we will happily connect you with a local church or more approriate venue.
+
+Should you participate in our community we will share our hope, prayers, worship and beliefs with you. These are not meant to descriminate and exclude, but instead to help support our community and deepen their faith.
+
+## Enforcement
+
+If you experience or witness unacceptable behavior—or have any other concerns—please report it by contacting us via [conduct@faithtech.com](mailto:conduct@faithtech.com). All reports will be handled with discretion.
```

Comment:
Two really minor things:
- `behavior-or` and `concerns-please` should have spaces around the dashes, like `behavior - or`
- are we going with Canadian or American spelling? (behavior vs behaviour)